### PR TITLE
Lobby Improvements

### PR DESCRIPTION
**Pull request by Cerebulon**
Ported and adapted from World Server.
:cl:
adds - Adds IC news to the lobby screen and newspapers, with new articles popping in lobby up when new.
adds - Adds map-specific introduction to lobby screen.
adds - Adds optional title var to newscaster postings.
adds - Added news articles from latter half of 2020.
tweak - Server News is now referred to as Game Updates, to avoid confusion.
/:cl:

The news popup operates off the Lore News Codex datum instead of a persistent newscaster, which was not ported.
Persistence elements in general were removed as they aren't currently desired.

example

NuTravis seems to be throwing a fit over HTML tags which are in fact fine. What gives, new guy?

### CODE REVIEW

File: code/modules/news/newspaper.dm [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+							dat+="<FONT SIZE=1>\[[MESSAGE.message_type] by <FONT COLOR='maroon'>[MESSAGE.author]</FONT>\]</FONT><BR><BR>"
+						dat+="</ul>"
+				if(scribble_page==curr_page)
+					dat+="<BR><I>There is a small scribble near the end of this page... It reads: \"[scribble]\"</I>"
+				dat+= "<BR><HR><DIV STYLE='float:left;'><A href='?src=\ref[src];prev_page=1'>Previous Page</A></DIV> <DIV STYLE='float:right;'><A href='?src=\ref[src];next_page=1'>Next Page</A></DIV>"
+			if(2) //Last page
+				for(var/datum/feed_channel/NP in news_content)
+					pages++
+				if(important_message!=null)
+					dat+="<DIV STYLE='float:center;'><FONT SIZE=4><B>Wanted Issue:</B></FONT SIZE></DIV><BR><BR>"
```
Comment (Schnayy) [inline code/modules/news/newspaper.dm:79]: Try just FONT instead of </FONT SIZE>

File: code/modules/news/news_init.dm [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+/datum/lore/news/New()
+	..()
+	spawn(50) //Give it a second or it gets fucky.
+		for(var/datum/feed_channel/F in news_network.network_channels)
+			if(F.channel_name == "Vir News Network")
+				station_newspaper = F
+				break
+	spawn(300) // Yes, again.
+		fill_codex_news()
+		to_world("I'm walkin here")
```
Comment (Atermonera) [inline code/modules/news/news_init.dm:20]: to_world()

File: code/modules/news/news_init.dm [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+/datum/lore/news/proc/fill_codex_news()
+	if(!news_network)
+		to_world("Load: Could not find newscaster network!")
```
Comment (Atermonera) [inline code/modules/news/news_init.dm:24]: These two should probably be `log_debug()`

File: code/modules/news/newspaper_layout.dm [hunk 4]
```diff
@@ -0,0 +1,41 @@
+/proc/get_newspaper_header(newspaper, var/list/subjects, bgcolor)
```
Comment (Atermonera) [inline code/modules/news/newspaper_layout.dm:1]: Why do these need to be global procs?

**Pull request merged**